docs/configuration.md

```diff
@@ -33,7 +33,7 @@ Off-heap memory requirement can be estimated as below:
 
 ```math
 numDirs = `celeborn.worker.storage.dirs`      # the amount of directory will be used by Celeborn storage
-bufferSize = `rss.worker.flush.buffer.size`   # the amount of memory will be used by a single flush buffer 
+bufferSize = `celeborn.worker.flusher.buffer.size`   # the amount of memory will be used by a single flush buffer 
```

Comment:
I appreciate your willingness. This configuration file should be deleted because we already achieved auto-generation docs.

Reply:
So just remove this file.

Reply:
oops, let me delete the configuration file.